src/actions/createUserActions.js

```diff
@@ -1,68 +1,83 @@
-let fs = require("fs");
-const { readFile, writeFile, maxId } = require("../helpers/helpers");
+// let fs = require("fs");
+// const { readFile, writeFile, maxId } = require("../helpers/helpers");
 
-let createUser = (req, res) => {
-  readFile("db/users.json")
-    .then(data => {
-      const users = JSON.parse(data);
-      let body = req.body;
+// let createUser = (req, res) => {
```

Comment:
When do you plan to delete this commented code?

This is very verbose: a lot of words(lines of code) that I have to read(scroll over) because they don't do anything.

If the main purpose is to see the history (what was the old code), you have the git for that (you can see file differences). Most editors support version comparison using the repository.

If the commented code should use as explanation, explain the changes before, in a TDC, or during the change (with PP) or after, using relevant git commit messages.

Reply:
Well, given the fact this wasn't their initial code, I think just for the courtesy of giving us the chance to re-factor our own code, until further update this should remain like this. Initially we asked for this file to not be included, but since it's the only example which can be given for the rest of the teams, we agreed to upload this in this version, we will re-write it without peeking.

Reply:
Agreed. That's fine (to use a branch as an example for the code). Don't create a PR with it.
This was a PR with enough review approvals to be accepted (and merged into master).